selinux/tabrmd.te

```diff
@@ -13,10 +15,16 @@ allow tabrmd_t self:unix_dgram_socket { create_socket_perms };
 
 dev_rw_tpm(tabrmd_t)
 logging_send_syslog_msg(tabrmd_t)
+sysnet_dns_name_resolve(tabrmd_t)
 
 optional_policy(`
     dbus_stub()
     dbus_system_domain(tabrmd_t, tabrmd_exec_t)
     allow system_dbusd_t tabrmd_t:unix_stream_socket rw_stream_socket_perms;
 ')
 
+tunable_policy(`tabrmd_can_network_connect',`
+    gen_require(`type unreserved_port_t;')
+
+    allow tabrmd_t unreserved_port_t:tcp_socket name_connect;
```

Comment:
This line should be replaced by: corenet_tcp_connect_all_unreserved_ports(tabrmd_t)